indigo/distribution.yaml

```diff
@@ -6893,6 +6893,16 @@ repositories:
       type: git
       url: https://github.com/tum-vision/lsd_slam.git
       version: master
+  lslidar_C16:
```

Comment:
Please remove the changes to `indigo/distribution.yaml` since Indigo reached EOL in April.

Reply:
Hi, I have removed changes in indigo/distribution.yaml.